src/python/pants/core/goals/lint.py

```diff
@@ -68,9 +69,12 @@ def create(
         *,
         strip_chroot_path: bool = False,
         report: Digest = EMPTY_DIGEST,
+        strip_formatting: bool = False,
```

Comment:
"strip formatting" doesn't really imply stripping colors to me.

Call a spade a spade? `strip_colors`?

Reply:
This is the name of the arg we use elsewhere too: https://github.com/pantsbuild/pants/blob/b1fdb7bbd705ae2287a8a0bc53405e03c57a8dc6/src/python/pants/core/goals/check.py#L55

I'd prefer to keep the current pattern for local consistency, and you can rename globally as separate work if you feel strongly about it.

Reply:
(In addition, I think the `colours.strip_color` function strips other terminal formatting escape codes, like bold and underline too, despite its name.)